docs/.custom_wordlist.txt

```diff
@@ -5,7 +5,7 @@ Github
 frictionless
 COU
 cou
-COUModel
+Model
```

Comment:
I don't think this need to be part of the custom wordlist

Reply:
Not sure, why global search did not found it. Thanks

Reply:
Ach, no I changed and it's not needed. Removing